### DISCUSSION TITLE
Can I make the chat window disappear from my screen after a certain period of time?

### DISCUSSION THREAD

**Discussion by icevoltt** [baffler/Transparent-Twitch-Chat-Overlay#284] [Q&A]
I am not currently broadcasting on Twitch, so I am broadcasting using a custom URL.

After several searches, I found and entered the CSS code that fades out the chat window, but the chat window still remains on my screen. 

This takes up the screen and causes inconvenience. 

Is it possible to make the chat window disappear over time?

The applied code is like this

body { 
    background-color: rgba(0, 0, 0, 0); 
    margin: 0px auto; 
    overflow: hidden; 
}

[class^="name_text"] {
    font-size: 15px;
    line-height: 1.2;
    font-weight: bold;
    padding: 3px, 3px;
}

[class^="live_chatting_message_text"] {
    font-size: 15px;
    line-height: 1.2;
    padding: 3px, 3px;
    font-weight: bold;
    color: rgb(255,255,255) !important;
}

img {
    vertical-align: middle;
    margin-top: 1px;
    margin-left: 0px;
    margin-right: 0px;
    width: 18px !important;
    height: 18px !important;
}

[class^="live_chatting_message_container"][class*="live_chatting_message_is_overlay"] > [class^="live_chatting_message_wrapper"] {
    line-height: normal;
    padding: 3px 3px;
}

.chat-message {
    animation: fadeOut 30s forwards;
}

@keyframes fadeOut {
    0% { opacity: 1; }
    100% { opacity: 0; }
}

**Comment by EuFontoura** [comment]
Try this

@keyframes fadeOut {
  0% { opacity: 1; }
  99% { opacity: 0; visibility: visible; }
  100% { opacity: 0; visibility: hidden; }
}
